presto-cli/src/main/java/com/facebook/presto/cli/ClientOptions.java

```diff
@@ -75,6 +75,9 @@
     @Option(name = "--user", title = "user", description = "Username")
     public String user = System.getProperty("user.name");
 
+    @Option(name = "--password", title = "password", description = "Password")
```

Comment:
We should have the option of prompting for the password rather than passing it on the command line. Maybe also allow using an environment variable?